src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -428,6 +432,54 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         return api.getUsersAPI(false).getAvatarUrlWithAccessToken(userId, file.id);
     };
 
+    renderAddTaskButton = () => {
+        const { isDisabled } = this.props;
+        const { isTaskFormOpen, currentUser } = this.state;
+        const closeTaskModal = () => this.setState({ isTaskFormOpen: false });
+        const openTaskModal = () => this.setState({ isTaskFormOpen: true });
+        const handleClickAdd = openTaskModal;
```

Comment:
What is the reason for reassigning/renaming these variables? I would have thought this would cause an eslint warning.